assets/tests/components/searchForm.test.tsx

```diff
@@ -252,21 +258,48 @@ describe("SearchForm", () => {
   })
 
   describe("filters", () => {
-    test("all filters are shown as inactive by default", () => {
-      const testDispatch = jest.fn()
-      render(
-        <StateDispatchProvider state={initialState} dispatch={testDispatch}>
-          <SearchFormFromStateDispatchContext />
-        </StateDispatchProvider>
-      )
+    describe("the selected property is checked by default", () => {
+      const propertyToSelector = {
+        all: allFilter,
+        vehicle: vehicleFilter,
+        operator: operatorFilter,
+        run: runFilter,
+        location: locationFilter,
+      }
+      const filters: SearchPropertyQuery[] = [
+        "all",
+        "vehicle",
+        "operator",
+        "run",
+        "location",
+      ]
+      test.each(filters)("%s checked", (selectedProperty) => {
+        const testDispatch = jest.fn()
+        render(
+          <StateDispatchProvider
+            state={{
+              ...initialState,
+              searchPageState: {
+                ...initialState.searchPageState,
+                query: emptySearchQueryFactory.build({
+                  text: "123",
+                  property: selectedProperty,
+                }),
+              },
```

Comment:
nit(non-blocking): would it be possible to use the `stateFactory` here instead of `initialState`?